src/Views/Auth/Auth.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { useState } from 'react/cjs/react.development';
+import AuthForm from '../../Components/Auth/AuthForm';
+import { signInUser } from '../../services/users';
+
+export default function Auth({ setCurrentUser }) {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState('');
+  const history = useHistory();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      // console.log('click');
+      const resp = await signInUser(email, password);
+      setCurrentUser(resp);
```

Comment:
I think we will want to setCurrentUser here as resp.email. Not entirely sure if this is causing netlify to break.